internal/pkg/pkg_test.go

```diff
@@ -511,3 +512,135 @@ func TestSubpackages_symlinks(t *testing.T) {
 	}
 	assert.Equal(t, []string{"subpkg"}, paths)
 }
+
+func TestFunctionConfigFilePaths(t *testing.T) {
```

Comment:
Functions follow data set kind of tests correct ? Why are we using pkgbuilder ? Frank mentioned that we would eventually move to dataset tests everywhere as a non-coder also can read and add tests.

Reply:
`pkgbuilder` is for unit test. Dataset test are for e2e tests.

Reply:
That is not the intent IMO. But we can clean this up later as part of overall migration.